### PR TITLE
Fix MMIO IOBinder

### PR DESCRIPTION
**Pull request by jwright6323**
**Related issue**: None

**Type of change**: bug fix

**Impact**: rtl change

**Release Notes**
- Fixes a bug in the MMIO port IO binder when a mem port is not present

### CODE REVIEW

File: generators/chipyard/src/main/scala/IOBinders.scala [hunk 1]
```diff
@@ -297,7 +297,7 @@ class WithAXI4MMIOPunchthrough extends OverrideLazyIOBinder({
   (system: CanHaveMasterAXI4MMIOPort) => {
     implicit val p: Parameters = GetSystemParameters(system)
     val clockSinkNode = p(ExtBus).map(_ => ClockSinkNode(Seq(ClockSinkParameters())))
-    clockSinkNode.map(_ := system.asInstanceOf[HasTileLinkLocations].locateTLBusWrapper(MBUS).fixedClockNode)
```
Comment (jerryz123) [inline generators/chipyard/src/main/scala/IOBinders.scala:300]: I thought the axi4 port takes its clock from the MBus, not the SBus?

Comment (jwright6323) [reply generators/chipyard/src/main/scala/IOBinders.scala:300]: It was on the sbus in the generated RTL I looked at. This seems to confirm, but I could be wrong:
https://github.com/chipsalliance/rocket-chip/blob/86a2f2cca699f149bcc082ef2828654a0a4e3f4b/src/main/scala/subsystem/Ports.scala#L147

**Review by jerryz123 — APPROVED** [review]

**Comment by jwright6323** [comment]
@jerryz123 do you know why the tests are getting canceled?

**Review by abejgonzalez — CHANGES_REQUESTED** [review]
As you can tell, CircleCI isn't working right now (I submitted a help ticket to get this resolved but it might take a while). Since GitHub Actions CI is still working, could you add your test to GH-A then we can merge?

**Comment by jwright6323** [comment]
Thanks for picking up the slack @abejgonzalez

**Pull request merged**